### ISSUE TITLE
Modify data extraction script

### ISSUE THREAD

**Issue by PedroPianna** [zetenyLevay/project2-1_group15_machine_learning#83]
#### This follows from issue #82. The task for this issue is to modify extract_mlagents_data.py to collect the hardware and time data generated by #71 and the PerformanceLogger.cs class. Closing this issue will mean we can extract all relevant training data, which will then be used to train the ML models

Edit: I've noticed some values aren't being retrieved from tensorbord, such as Learning Rate, Policy/Beta, Entropy, Epsilon, etc. Please make sure to log everything that isn't a constant (a.k.a everything that isn't from the YAML file or static.csv)
